diff --git a/project.cpp b/project.cpp
index 81f6fd4..d05d0d1 100644
--- a/project.cpp
+++ b/project.cpp
@@ -1,4 +1,4 @@
-#include "al/al_console.h"
+ #include "al/al_console.h"
 #include "al/al_math.h"
 #include "al/al_gl.h"
 #include "al/al_mmap.h"
@@ -12,7 +12,7 @@ unsigned int VBO;
 unsigned int instanceVBO;
 
 float vertices[] = {
-    -0.5f, -0.5f, 0.0f,
+    -0.3f, -0.5f, 0.0f,
      0.5f, -0.5f, 0.0f,
      0.0f,  0.9f, 0.0f
 };
